ext/ed.h

```diff
@@ -27,7 +27,7 @@ class SslBox_t; // forward reference
 #endif
 
 bool SetSocketNonblocking (SOCKET);
-
+bool SetFdCloexec (int);
```

Comment:
Yeah, I was reluctant to do that since we apply it to non-socket FDs, too. Happy to perform the rename if you say that's ok (-: